src/components/WorkOrders/SummaryCloseWorkOrder.test.js

```diff
@@ -35,8 +35,8 @@ describe('SummaryCloseWorkOrder component', () => {
       <SummaryCloseWorkOrder
         reference={props.reference}
         notes={props.notes}
-        time={props.time}
-        date={props.date}
+        completionDate={props.completionDate}
```

Comment:
I know it's no new addition but I feel the title of this test makes me think there should be an assertion to verify these rows don't exist but there are no assertion other than a snapshot comparison to what's rendered